src/services/marp_service.py

```
@@ -0,0 +1,65 @@
+    def generate_html(self, output_filename="slides.html", theme=None):
+        return self._generate(self.OutputType.HTML, output_filename, theme=theme)
+
+    def generate_png(self, output_filename="slides.png", theme=None):
+        return self._generate(self.OutputType.PNG, output_filename, theme=theme)
+
+    def generate_pptx(self, output_filename="slides.pptx", theme=None):
+        return self._generate(self.OutputType.PPTX, output_filename, theme=theme)
+
+    def _generate(self, output_type, output_filename, theme=None):
+        if not self.output_dir:
+            raise ValueError("Output directory must be set for generation.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)